src/app/(layout)/new/routes/row.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { Badge } from '@/components/ui/badge';
+import { TableCell, TableRow } from '@/components/ui/table';
+import { Tooltip } from '@/components/ui/tooltip';
+import { formatMinutes } from '@/lib/time';
+import { useRouter } from 'next/navigation';
+
+type RouteResult = {
+	id: number;
+	origin_iata: string;
+	destination_iata: string;
+	origin_name: string;
+	destination_name: string;
+	average_duration: number;
+	aircraft_short_names: string;
+};
+
+export default function Row({
+	aircraft,
+	route,
+}: { aircraft: string[]; route: RouteResult }) {
+	const router = useRouter();
+	return (
+		<TableRow
+			className="cursor-pointer"
+			onClick={() => router.push(`/new/routes/${route.id}`)}
```

Comment:
extract to use useCallback, `const navigate = useCallback(() => { router.push(...) }, []);`